src/tests/suite/daos_drain_simple.c

```diff
@@ -1003,6 +1003,8 @@ run_daos_drain_simple_test(int rank, int size, int *sub_tests,
 {
 	int rc = 0;
 
+	FAULT_INJECTION_REQUIRED();
+
```

Comment:
So this is the reason for the cmocka reporting failure on master. Instead of putting this line here, it should go into each individual sub test so the results are still reported. Doing so should also resolve the build issues.

The problem is that FAULT_INJECTION_REQUIRED skips/returns, so the below code never populates the cmocka tests

Reply:
Thanks